drivers/netdev_ieee802154_submac/netdev_ieee802154_submac.c

```diff
@@ -354,6 +354,14 @@ static int _init(netdev_t *netdev)
     /* This function already sets the PAN ID to the default one */
     netdev_ieee802154_reset(netdev_ieee802154);
 
+#if IS_USED(MODULE_IEEE802154_SECURITY)
+    const ieee802154_radio_cipher_ops_t *cipher_ops;
+    if ((cipher_ops = ieee802154_radio_get_cipher_ops(&submac->dev))) {
+        netdev_ieee802154->sec_ctx.dev.cipher_ops = cipher_ops;
+        netdev_ieee802154->sec_ctx.dev.ctx = &submac->dev;
```

Comment:
Since it was basically `submac->dev.priv` before, benpiccos recent suggestions would be necessary.